.travis.yml

```diff
@@ -72,6 +73,7 @@ install:
 - source scripts/setup_gae.sh || exit 1
 
 script:
+- if [ $RUN_PATTERN_CHECK == 'true' ]; then python scripts/run_pattern_check.py; fi
```

Comment:
The matrix and scripts are not in similar order. Shall i change the entire order ?